src/components/Core/Player/Player.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import styles from './Player.scss'
 import {artworkForMediaItem} from "../common/Utils";
+import _ from 'lodash';
```

Comment:
Lets import only the `debounce` function of lodash

Reply:
Resolved.